CITATION.cff

```diff
@@ -0,0 +1,73 @@
+cff-version: "1.2.0"
```

Comment:
What is this version exactly? The version of soundata? Don't think so, right?

Reply:
the .cff file was autogenerated by JOSS. It is the latest version of CITATION.cff https://github.com/citation-file-format/citation-file-format/blob/main/CHANGELOG.md#120---2021-05-31

Reply:
right